### ISSUE TITLE
Fatal error in OrganizationFieldsApi->getOrganizationFields() 

### ISSUE THREAD

**Issue by knutselsmurf** [pipedrive/client-php#112]
~~~
Fatal error: Uncaught InvalidArgumentException: Invalid value for enum '\Pipedrive\Model\FieldTypeAsString', must be one of: 'address', 'date', 'daterange', 'double', 'enum', 'monetary', 'org', 'people', 'phone', 'set', 'text', 'time', 'timerange', 'user', 'varchar', 'varchar_auto', 'visible_to' in /vendor/pipedrive/pipedrive/lib/ObjectSerializer.php:375
~~~
Retrieving indivudual fields via  OrganizationFieldsApi->getOrganizationField($id)  works fine

**Comment by Reivilo85k** [comment]
This is a duplicate of https://github.com/pipedrive/client-php/issues/97

**Comment by s-pogoda** [comment]
Hello @knutselsmurf ,

I'm writing to let you know, that the issue with getting Pipedrive entity fields is now fixed in the `7.1.0` version

Sorry for the delayed replay
